_posts/2019-10-30-upgrade-reminder.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Make sure you are ready for the Bitcoin Cash network upgrade
```

Comment:
Should we put a date reminder as a subtitle? Something like `Upgrade activates on Nov. 15th`?